core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/WorkspaceComponent.scala

```diff
@@ -161,8 +161,8 @@ trait WorkspaceComponent {
       }
     }
 
-    def getTags(queryString: Option[String], limit: Option[Int] = None): ReadAction[Seq[WorkspaceTag]] = {
-      val tags = workspaceAttributeQuery.findUniqueStringsByNameQuery(AttributeName.withTagsNS, queryString, limit).result
+    def getTags(queryString: Option[String], limit: Option[Int] = None, ownerIds: Option[Seq[UUID]] = None): ReadAction[Seq[WorkspaceTag]] = {
```

Comment:
Is there Swagger documentation to update for this change?

Reply:
I've updated the description here: 0616372fb